packages/@aws-cdk/aws-rds/lib/cluster-engine.ts

```diff
@@ -76,11 +106,13 @@ interface ClusterEngineBaseProps {
   readonly multiUserRotationApplication: secretsmanager.SecretRotationApplication;
   readonly defaultPort?: number;
   readonly engineVersion?: EngineVersion;
+  readonly features?: EngineFeatures;
 }
 
 abstract class ClusterEngineBase implements IClusterEngine {
   public readonly engineType: string;
   public readonly engineVersion?: EngineVersion;
+  public readonly features?: EngineFeatures;
```

Comment:
Any reason this is `public`? I think it can safely be `private`.

Reply:
I think private is fine for now. I was originally thinking it might be useful for other features but we can always walk through that door if it makes sense to.